### ISSUE TITLE
Issues with power for audio on battery

### ISSUE THREAD

**Issue by stonevil** [stonevil/Razer_Blade_Advanced_early_2019_Hackintosh#10]
Issues with power for audio on battery

https://www.tonymacx86.com/threads/guide-razer-blade-15-2018-detailed-install-guide-high-sierra-10-13-6-17g2208-17g5019.264017/post-1988609

**Comment by k-sym** [comment]
@stonevil - can you confirm symptoms?
[edit] - ignore, just seen your post on TM. Looking forward to details of fix :D

**Comment by stonevil** [comment]
Sorry, temporary without Razer Blade. I will back to fix this issue when will receive my new Razer.

P.S. Calm down. My Razer is alive. No issues.

**Comment by doubleyoustew** [comment]
I'm using alc-layout 3 now and it seems to work better. I didn't have audio problems for about four days. Also alc-delay might help too.

**Comment by ronanarm** [comment]
@doubleyoustew Tried alcid=3 and alcdelay=3000, I'm still getting the issue, just keeping this thread updated.

edit: headphone jack seems to work all the time, just the speakers seem to have issues.

**Comment by doubleyoustew** [comment]
Yeah unfortunately it's not completely fixed, although I feel like with alc layout 3 it's much more rare and if it happens shutting down and stating back up fixed it for me, even on battery which wasn't the case before. Only had the problem once since switching. There are other layouts that you could try in the AppleALC repo. I don't have the time to go though all of them at the moment and it's a little tricky since the problem occurs so rarely for me now.

The only time the speakers didn't work with layout 3 was when I rebooted from Windows into macOS without shutting down first. Maybe that is reproducible but I don't know. The jack and mic however always seem to work for me too.
